crates/lox_core/src/bodies/cip/xy06.rs

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2023. Helge Eichhorn and the LOX contributors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+mod amplitudes;
+mod luni_solar;
+mod planetary;
+mod polynomial;
+
+use crate::bodies::fundamental::iers03::{
+    general_accum_precession_in_longitude_iers03, mean_moon_sun_elongation_iers03,
+};
+use crate::bodies::{Earth, Jupiter, Mars, Mercury, Moon, Neptune, Saturn, Sun, Uranus, Venus};
+use crate::math::arcsec_to_rad;
+use crate::time::intervals::TDBJulianCenturiesSinceJ2000;
+use crate::types::Radians;
+
+/// A convenient type for performing batch mathematical operations on X and Y components. This
+/// type may change or become unexported as the needs of upstream components become clearer.
+pub type XY = [f64; 2];
```

Comment:
In `/home/matzipan/Workspace/lox/crates/lox_core/src/ephemeris/daf_spk/api.rs` type `Position` is a triple. Would it make sense to use a touple here?